test/remote_coffee_slack/members_test.rb

```diff
@@ -17,6 +17,7 @@ def test_select_coffee_mates_with_members
       RemoteCoffeeSlack.config.members_per_group = 4
```

Comment:
@JuanCrg90  looks like this test did not update configuration back to 2 members per group.
That's why the previous one was failing when run after this one

Reply:
Could you open an issue for this please?

Reply:
I fixed it here https://github.com/Calzada-Code/remote-coffee-slack/pull/13/commits/da63629dc39e60a02b580fed0ff751ba2d7715a3